src/compiler/types.ts

```diff
@@ -6022,12 +6022,10 @@ export const enum NodeCheckFlags {
     ContainsCapturedBlockScopeBinding        = 1 << 13,  // Part of a loop that contains block scoped variable captured in closure
     CapturedBlockScopedBinding               = 1 << 14,  // Block-scoped binding that is captured in some function
     BlockScopedBindingInLoop                 = 1 << 15,  // Block-scoped binding with declaration nested inside iteration statement
-    ClassWithBodyScopedClassBinding          = 1 << 16,  // Decorated class that contains a binding to itself inside of the class body.
-    BodyScopedClassBinding                   = 1 << 17,  // Binding to a decorated class inside of the class's body.

```

Comment:
Should we shift the other bits down or is it no big deal to leave them empty until later?

Reply:
this is a driveby cleanup, right? I don't see any removal of BodyScopedClassBinding in the rest of the PR.

Reply:
All references to `BodyScopedClassBinding` were removed in a prior PR, but I neglected to remove the flag. I can fix up the bits used as well.